test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,39 +1,52 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.Test;
 
 import java.util.Arrays;
-import java.util.List;
 
-import org.junit.Test;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class UtilsTest {
 
 
     @Test
-    public void elementsAreUnique() throws Exception {
+    public void elementsAreUnique() {
```

Comment:
It is recommended to separate out testing different functionalities into different JUnit tests.